From dfe68a953f8518ad1b6d9ce6fdcdbbbf2fb3eed5 Mon Sep 17 00:00:00 2001
From: Oliver Brown <oliver.brown@nxp.com>
Date: Wed, 24 Oct 2018 12:47:08 -0500
Subject: [PATCH 4927/5242] MLK-20063 drm: imx: dp: Correct iMX8M PHY
 initialization

commit  dfa0a7eeefe5ad9aa88e5a3c3605cf22fb5d68ac from
https://source.codeaurora.org/external/imx/linux-imx.git

The DisplayPort PHY initialization does not need the additional
loop gain enabled.

Signed-off-by: Oliver Brown <oliver.brown@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/gpu/drm/imx/hdp/API_AFE_mcu2_dp.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/imx/hdp/API_AFE_mcu2_dp.c b/drivers/gpu/drm/imx/hdp/API_AFE_mcu2_dp.c
index 900f126..08904cc 100644
--- a/drivers/gpu/drm/imx/hdp/API_AFE_mcu2_dp.c
+++ b/drivers/gpu/drm/imx/hdp/API_AFE_mcu2_dp.c
@@ -518,7 +518,7 @@ void afe_init_t28hpc(state_struct *state,
 	val = val | 0x0008;
 	Afe_write(state, PHY_PMA_CMN_CTRL1, val);
 
-	Afe_write(state, CMN_DIAG_PLL0_TEST_MODE, 0x0022);
+	Afe_write(state, CMN_DIAG_PLL0_TEST_MODE, 0x0020);
 	Afe_write(state, CMN_PSM_CLK_CTRL, 0x0016);
 
 	if (refclk == REFCLK_24MHZ) {
-- 
1.7.9.5

